src/coreclr/clrdefinitions.cmake

```diff
@@ -176,7 +176,9 @@ endif (CLR_CMAKE_HOST_UNIX AND CLR_CMAKE_TARGET_UNIX)
 if (FEATURE_ENABLE_NO_ADDRESS_SPACE_RANDOMIZATION)
   add_definitions(-DFEATURE_ENABLE_NO_ADDRESS_SPACE_RANDOMIZATION)
 endif(FEATURE_ENABLE_NO_ADDRESS_SPACE_RANDOMIZATION)
-add_definitions(-DFEATURE_SVR_GC)
+if (NOT CLR_CMAKE_HOST_ANDROID)
+  add_definitions(-DFEATURE_SVR_GC)
```

Comment:
Shold we do the same for the other platforms contained in the `FEATURE_STANDALONE_GC` above? Unclear if server GC makes sense for any of the mac related skus either.

Reply:
> Unclear if server GC makes sense for any of the mac related skus either.

Server GC does make sense for mac skus. For example, server GC is enabled for the compiler server (`VBCSCompiler.dll`) since it is large heavily parallel app. Removing server GC on mac would regrade developer experience on mac.